src/parent/parent.js

```diff
@@ -862,8 +863,18 @@ export function parentComponent<P, X, C>({
       })
       .then((isClosed) => {
         if (!cancelled) {
-          if (isClosed) {
-            return close(new Error(`Detected ${context} close`));
+          const isCurrentContainerClosed: boolean =
+            // $FlowIssue
+            currentContainer && isElementClosed(currentContainer);
```

Comment:
Are you able to remove the `$FlowIssue` statement by using Boolean() like so?
```suggestion
            Boolean(currentContainer && isElementClosed(currentContainer));
```